packages/utils/src/file/convertImageToBlob/index.ts

```diff
@@ -0,0 +1,54 @@
+const TYPE_TO_FORMAT_MAPPER = {
+  png: 'image/png',
+  jpg: 'image/jpg',
+  jpeg: 'image/jpeg',
+  webp: 'image/webp',
+} as const;
+
+const createBlobFromCanvas = (canvas: HTMLCanvasElement, format: string) => {
+  return new Promise((resolve, reject) => {
+    canvas.toBlob((blob) => {
+      if (blob) {
+        resolve(blob);
+      } else {
+        reject(new Error(`Failed to create blob with format ${format}`));
+      }
+    }, format);
+  });
+};
+
+export const convertImageToBlob = (
+  url: string,
+  imageType: keyof typeof TYPE_TO_FORMAT_MAPPER = 'png',
+) => {
+  return new Promise((resolve, reject) => {
+    const img = new Image();
+
+    img.onload = async () => {
+      const canvas = document.createElement('canvas');
+      canvas.width = img.width;
+      canvas.height = img.height;
+
+      try {
+        const ctx = canvas.getContext('2d');
+        if (!ctx) throw new Error('Failed to get 2d context');
+
+        ctx.drawImage(img, 0, 0);
+
+        const result = await createBlobFromCanvas(
+          canvas,
+          TYPE_TO_FORMAT_MAPPER[imageType],
+        );
+        resolve(result);
+      } catch (error) {
+        reject(error);
+      }
```

Comment:
```suggestion
      } catch (error: any) {
        console.error(`Failed to load image. message: ${error.mesaage}`)
        reject(error);
      }
```
기존 error를 그래도 reject해주는 것과 더불어 콘솔 에러를 추가해주는 것은 어떨까요!?

Reply:
오오 놓쳤던 부분인데 너무 좋은것같습니다! 위에 코멘트 주신 부분과 같이 반영해두겠습니다! 👍